packages/blade/src/components/Button/BaseButton/__tests__/__snapshots__/BaseButton.native.test.tsx.snap

```diff
@@ -65,42 +67,60 @@ exports[`<BaseButton /> should render button with default properties 1`] = `
       },
     ]
   }
+  type="button"
```

Comment:
This is being passed down now because of spreading props. Imagine unintended props like `as`  being passed down which might break a lot of things 😅